src/libcrun/utils.h

```diff
@@ -422,6 +422,8 @@ get_process_exit_status (int status)
 uid_t get_overflow_uid (void);
 gid_t get_overflow_gid (void);
 
+void consume_trailing_slashes (char *path);
```

Comment:
We could move it right below the following:

- https://github.com/containers/crun/pull/1517/files#diff-e84345e78d59b60cc2aadd7662467f8959e7baa95c1ec9b58ad04f7196ae2273R228-R234